README.adoc

```diff
@@ -119,6 +119,14 @@ can act as starting points or examples:
  provider such as https://stormpath.com/[Stormpath], Active Directory,
  https://developers.google.com/identity/[Google], http://openid.net/[OpenID], etc.
 
+ https://github.com/SMARTRACTECHNOLOGY/devkit-simple-service-example[Create Your Own Microservice]::
```

Comment:
I see it differently: The DevKit is the starting point for external users/customers, and it provides them with a collection of things to do with the DevKit or based on the DevKit. It's not just documenting how to work with the DevKit, it's also intended to cover more advanced topics.

Extending the DevKit by an additional service is part of this. It's advanced, but it belongs to the set of DevKit guides. We could even consider making this repository a sub-module of the DevKit repository.

I also think that the tutorial is not that dependent on the the code in the repository. It's mostly maven-related, and therefore could fit into the general DevKit repository.